react/features/base/react/components/native/AbstractPagedList.js

```diff
@@ -137,14 +142,20 @@ export default class AbstractPagedList extends Component<Props, State> {
             pageIndex: validatedPageIndex
         });
 
+        const { dispatch, onSelectPage, pages } = this.props;
```

Comment:
We usually extract from an instance property such as `this.props` into a local variable if the value is to be used at least twice; otherwise, I see very little point in pulling multiple values, especially ones which are conditionally used.